src/Symfony/Component/Serializer/Annotation/SerializedPath.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Annotation;
+
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+
+/**
+ * Annotation class for @SerializedPath().
+ *
+ * @Annotation
+ *
+ * @NamedArgumentConstructor
+ *
+ * @Target({"PROPERTY", "METHOD"})
+ *
+ * @author Tobias Bönner <tobi@boenner.family>
+ */
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
+final class SerializedPath
+{
+    public function __construct(
+        private array $serializedPath
```

Comment:
What about accepting a string as well, and then casting the string to an array? Such as it's done for the `Groups` attribute?